bin/lib/git.py

```diff
@@ -1,14 +1,14 @@
 import subprocess
 
 
-def current_branch():
-    return __call(["git", "branch", "--show-current"])
+def has_gh_cli():
+    try:
+        return __call(["gh", "auth", "status"]) != "You are not logged into any GitHub hosts. Run gh auth login to authenticate."
+    except:
+        return False
 
-def clean_status():
-    return __call(["git", "status", "--porcelain"]) == ""
-
-def current_commit():
-    return __call(["git", "rev-parse", "--short", "HEAD"])
+def current_master_commit():
+    return __call(["git", "rev-parse", "--short", "origin/master"])
```

Comment:
Using the origin instead of trying to enforce being up-to-date locally is a good idea.